### PR TITLE
Fixes #13914 - host and fact importing refactored

### PR DESCRIPTION
**Pull request by lzap**
This was refactored in core: https://github.com/theforeman/foreman/pull/3242

### CODE REVIEW

File: app/models/host/discovered.rb [hunk 1]
```diff
@@ -144,7 +144,8 @@ def proxy_url
 
   def refresh_facts
     facts = ::ForemanDiscovery::NodeAPI::Inventory.new(:url => proxy_url).facter
-    self.class.import_host_and_facts facts
+    self.class.import_host facts
+    import_facts facts
```
Comment (dLobatog) [inline app/models/host/discovered.rb:148]: Aren't facts imported in line 57 (`host.import_facts(facts)`)?

Comment (lzap) [inline app/models/host/discovered.rb:148]: This is explicit call when user clicks on `Refresh facts` button.

**Comment by lzap** [comment]
Ping?

**Comment by dLobatog** [comment]
Merged as 57cd4c76414a6f1dc830a5d2283b0b24a9690319, thanks @lzap!

**Pull request closed without merging**